docs/tutorials/setup-cra.rst

```diff
@@ -77,24 +68,7 @@ Install
       # or using Yarn
       yarn extract
 
-   There should be no error and you should see output similar following:
```

Comment:
I have removed this because I noticed stats are commented out in extract command. Then I noticed a similar table is in many places in the docs. I guess it will have to be updated globally when stats are fixed.

Reply:
Yeah, I've forgot to fix stats. Now with multiple catalogs it isn't trivial :)